.commitlintrc.json

```diff
@@ -21,7 +21,7 @@
         "external",
         "file",
         "helm",
-	"k8sutil",
+        "k8sutil",
```

Comment:
@obnoxxx I noticed that one of your PRs added a tab here where the file uses spaces. Please update your editor so that this doesn't continue happening.

I have my editor set to show whitespace symbols, which helps me a lot, and is how I noticed this issue. Alternately, you might look into an extension which does a better job at auto-detecting space/tab usage in files and automatically configures your editor to use the correct one.

<img width="207" alt="image" src="https://github.com/rook/rook/assets/3989526/5794c44a-bd7d-41a5-b0d2-c07cb0287be1">